apps/frontend/src/components/global/HelpModal.vue

```diff
@@ -19,7 +19,7 @@
           <a href="htps://www.inspec.io">InSpec</a> profile execution (using
           InSpec's JSON reporter) or the results from
           <a href="https://github.com/mitre/heimdall_tools">Heimdall Tools</a>
```

Comment:
Thanks for the PR! Please know that Heimdall Tools has been ported to the [SAF CLI](https://github.com/mitre/saf) and this should reflected here.